extensions/wikia/EmergencyBroadcastSystem/EmergencyBroadcastSystemHooks.class.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+class EmergencyBroadcastSystemHooks extends WikiaController {
+	public function onBeforePageDisplay( OutputPage $out, $skin ) {
+		$app = F::app();
+		$assetsManager = AssetsManager::getInstance();
+
+		if ( $app->checkSkin( 'oasis' ) ) {
+			$scss = 'emergency_broadcast_system_scss';
+
+			foreach ( $assetsManager->getURL( $scss ) as $url ) {
+				$out->addStyle( $url );
```

Comment:
There should be easier way to add CSS to the page. Also - consider adding your new CSS to some already existing group - so there will not be a need to additional HTTP request.